openedx/core/lib/xblock_builtin/xblock_discussion/static/js/discussion_classes.js

```diff
@@ -0,0 +1,23 @@
+// Add RequireJS definitions for each discussion class
+
```

Comment:
@itsjeyd Two suggestions:

1. This looks like a requireJS shim - might be good to reflect it in the file name.
2. Add TODO/task to refactor discussions JS into using RequireJS? Actually this might not be what we want - I'm not sure is there any support for RequireJS in Apros; if there are no such support this is not an option (at least until we add it).